pkg/types/validation/installconfig_test.go

```diff
@@ -1399,6 +1400,42 @@ func TestValidateInstallConfig(t *testing.T) {
 				return c
 			}(),
 		},
+		{
+			name: "valid baseline capability set",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "v4.11"}
+				return c
+			}(),
+		},
+		{
+			name: "invalid baseline capability set specified",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "notValid"}
+				return c
+			}(),
+			expectedError: `"notValid": unknown baseline capability set specified`,
```

Comment:
nit: here's another place where we could help the user out with a list of options.  But probably a rare enough problem that I don't think we'd be fine as you have it too.

Reply:
Fixed in a10be333d